### ISSUE TITLE
Define a specific destination for each user using userfile

### ISSUE THREAD

**Issue by Trixpua** [rarcega/instagram-scraper#156]
Is possible define a custom destination folder for each user in a userfile?

I tried to make a script to run, specifing a custom folder for each user without userfile, but after scrap a few users, I get a problem with login.

I think that is interesting to implement this function, in user file can be specified a username/pathtosave or username@pathtosave, something like that, so can save in a path with a custom name, not only the username.

**Comment by gvkalra** [comment]
I believe it's possible to write a custom wrapper script using the following switches:

```
--filename    -f    Path to a file containing a list of users to scrape.

--destination -d    Specify the download destination. By default, media will 
                    be downloaded to <current working directory>/<username>.
```

**Comment by Trixpua** [comment]
Yes, but this way, I just change the parent path, if I want a custom path to each user, I just able to use the switch "-n", that keep the user name, I think will be interesting if a able to change this children path name, too.
I am newbie in python, but will try change the code, to achieve this result.

**Comment by alexiscoutinho** [comment]
@Trixpua Did you manage to change the code? If yes, would you consider making a PR?
